libs/auth/src/openssl.ts

```diff
@@ -200,16 +200,16 @@ export function verifyFirstCertWasSignedBySecondCert(
 }
 
 /**
- * Verifies a challenge signature against an original challenge using a public key (in PEM format).
+ * Verifies a message signature against an original message using a public key (in PEM format).
```

Comment:
A challenge is one kind of message - shifted to the more general term in prep for new upcoming uses of this function